app/controllers/home_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class HomeController < ApplicationController
+  def index
+    render
+  end
```

Comment:
We can do it without `render` 🚀 
```suggestion
  def index; end
```